### PR TITLE
Added customization to Rating component

### PR DESCRIPTION
**Pull request by dijs**
https://moovweb.atlassian.net/browse/PRPL-1864

- Added custom icon props for full, half, and empty
- Added fillEmpty option prop

### CODE REVIEW

File: packages/react-storefront/src/Rating.js [hunk 1]
```diff
@@ -21,10 +21,12 @@ export const styles = theme => ({
       color: theme.palette.rating,
       height: '16px',
       width: '16px',
-      marginLeft: '-2px',
```
Comment (markbrocato) [inline packages/react-storefront/src/Rating.js]: Why was this removed?  This will affect the look of the component, won't it?

File: packages/react-storefront/src/Rating.js [hunk 2]
```diff
@@ -65,29 +67,59 @@ export default class Rating extends Component {
...
     for (let i=1; i<=5; i++) {
       if (value == null || value >= i) {
-        stars.push(<Star key={i}/>)
+        stars.push(<IconFull key={i} />);
       } else if (value >= i - 0.5) {
-        stars.push(<StarHalf key={i}/>)
+        stars.push(<IconHalf key={i}/>)
       } else {
-        stars.push(<StarBorder key={i}/>)
+        if (fillEmpty) {
```
Comment (markbrocato) [inline packages/react-storefront/src/Rating.js]: Can we bump this if up one level to an `else if` and an `else`?

File: packages/react-storefront/src/Rating.js (hunk 1 above)
Comment (dijs) [reply packages/react-storefront/src/Rating.js]: I checked, it barely affects the default star icon, and it fixes any other custom icon we would use. The custom icons were overlapping each other.

File: packages/react-storefront/src/Rating.js (hunk 2 above)
Comment (dijs) [reply packages/react-storefront/src/Rating.js]: Done
